include/deal.II/integrators/l2.h

```diff
@@ -50,6 +50,11 @@ namespace LocalIntegrators
      * integrals \f[ \int_F uv\,ds \quad \text{or} \quad \int_F \mathbf u\cdot
      * \mathbf v\,ds \f]
      *
+     * @param M: The mass matrix matrix obtained as result.
+     * @param fe: The FEValues object describing the local trial function
+     * space. #update_values and #update_JxW_values must be set.
+     * @param factor: A constant that multiplies the mass matrix.
```

Comment:
Excellent, and thanks for adding this. We really appreciate the addition of documentation, large and small!

I'm reluctant to ask because it's so unnecessary, but I'll do that anyway: The way doxygen formats these `@param` statements is to offset the name of the argument (here, `M`, `fe`, `factor`) from the text that follows. Here is an example: https://dealii.org/developer/doxygen/deal.II/namespaceVectorTools.html#a2bdac18dce278e671f45bdcbf501b6e7

So the colon is not necessary and is likely going to look a bit funny after doxygen renders it. Would you mind removing this again?

In any case, many thanks again for the addition. I hope (*we* hope!) for many more patches from you!

Reply:
Whoops, should be good now!